public/index.js

```diff
@@ -1,17 +1,23 @@
 const resultElement = document.querySelector(".gpt-response");
```

Comment:
I'd probably group the` querySelector() `and `querySelectorAll() `together. I know I'm being pedantic here, but a senior developer told me it's kind of an industry standard as it makes the codebase cleaner and easier to read :-).


```
const resultElement = document.querySelector(".gpt-response");
const lactoseIntolerant = document.querySelector(".lactose-intolerant");
const buttons = document.querySelectorAll("button");
```

Reply:
No worries.